src/pages/Accounts/AccountDetailsPage.vue

```diff
@@ -54,13 +54,15 @@ onMounted(async () => {
   }
 });
 
-// use account's assets from payload instead or replace with fetching account's assets method
+// TODO: use assets from account payload instead
+//  or replace with fetching account's assets method
 const assetsTable = useTable(http.fetchAssetDefinitions);
```

Comment:
Asset definitions do not belong to accounts anyway. They belong to their domains.

Asset = asset definition + account.

```suggestion
const assetsTable = useTable(http.fetchAssets);
```

Reply:
✅